{
    "name": "libcups",
    "ecosystem": "pypi",
    "campaign": "2025-10-bugdotexe",
    "affected_versions": [
        "2.3.2",
        "2.4.2"
    ],
    "campaign_assigned_at": "2025-10-29T22:05:43.267890+00:00",
    "campaign_data_modified_at": "2025-10-30T07:14:26.214706+00:00"
}